src/assets/Enums.ts

```diff
@@ -11,7 +11,11 @@ export enum HTTPRESPONSE {
     RESOURCE_NOT_FOUND = "No resources match the search criteria.",
     INTERNAL_SERVER_ERROR = "Internal Server Error",
     MORE_THAN_ONE_MATCH = "The provided partial VIN returned more than one match.",
-    NO_STATUS_UPDATE_REQUIRED = "No status update required"
+    S3_ERROR = "Upload on S3 failed",
+    S3_DOWNLOAD_ERROR = "Cannot download document from S3",
```

Comment:
I see that you removed the code related to S3 after rebasing with develop. Shouldn't you remove these also?

Reply:
fixed